### PR TITLE
created Meeting minutes

### PR DESCRIPTION
**Pull request by TheFriendbesto**
Created meeting minutes folder, an entry for our first group meeting and a meeting minutes template for further entries.

Not using git from the command line so I am not sure how to use the .gitmessage. 
Sorry! I will need to figure out how to use it.

(RM - This addresses issue #6))

### CODE REVIEW

File: meeting_minutes/1-13-20.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+## Tasks discussed or worked on:
+[comment]: <> (Brief descripiton of talking topics, ideas for further exploration, and questions to follow up on)
+
+### Topics
+* Decided on a langauge for the main compiler Project. Congrats Python.
+* Set up main form of communication for team through Riot.
+* Created Git Repo, decided on commit style and started creating files to check push and merge credentials.
+
+### Research and Development
+* Look into Yap and other python formatters for cleaner code styles and for error checking.
```
Comment (Wmyers559) [inline meeting_minutes/1-13-20.md]:
```suggestion
* Look into YAPF and other python formatters for automatic code styling
* Suggested utilization of Pyre for type-checking
* Suggested utilization of CircleCI or similar for continuous testing/integration
* Suggested utilization of PyLint for code linting
```

File: meeting_minutes/1-13-20.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+### Questions
+* None as of yet.
+
+## Future Road map and issues:
+[comment]: <> (Please Link issues as they are created)
+
+* Set up CI #1 : 1/17/2020 : [Unknown]
+* Set up Pyer #2 : 1/17/2020 : [Unknown]
```
Comment (Wmyers559) [inline meeting_minutes/1-13-20.md]:
```suggestion
* Set up Pyre #2 : 1/17/2020 : [Reece]
```

File: meeting_minutes/1-13-20.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+* Set up YAPF #3 : 1/17/2020 : [Riley]
+* Set up meeting minutes #6 : 1/17/2020 : [Jeriah]
+* Set up Pylint #7 : 1/17/2020 : [Reece]
+* Set up Pylint #11 : 1/17/2020 : [unknown]
```
Comment (Wmyers559) [inline meeting_minutes/1-13-20.md]:
```suggestion
* Set up Pytest #11 : 1/17/2020 : [unknown]
```

**Comment by DragonAxe** [comment]
Looks good to me with Riley's suggested changes.

File: meeting_minutes/1-13-20.md (hunk 3 above)
Comment (TheFriendbesto) [reply meeting_minutes/1-13-20.md]: Updated and made the requested changes.

**Comment by DragonAxe** [comment]
Looks like you'll need to `git pull --rebase origin/master`. It also looks like there's a merge commit in the history of this branch, so you may need to squash your commits as well.

**Pull request merged**